client/src/components/AlumniDirectory.js

```diff
@@ -142,9 +143,9 @@ export default class AlumniDirectory extends Component {
                             </Card.Header>
                                             
                             <Card.Meta>{post.profession}</Card.Meta>
-                            <Card.Description>College: {post.college}</Card.Description>
-                            <Card.Description>Location: {post.location}</Card.Description>
-                            <Card.Description>Company: {post.company}</Card.Description>
+                            <Card.Description>College: {post.collegeName}</Card.Description>
+                            <Card.Description>Location: {(post.city && post.country) && `${post.city} (${post.country})`}</Card.Description>
```

Comment:
location is now split over country and city, currently only country is searchable